include/interface/blas1_interface.h

```diff
@@ -243,6 +243,45 @@ typename executor_t::policy_t::event_t _rotm(
     executor_t &ex, index_t _N, container_0_t _vx, increment_t _incx,
     container_1_t _vy, increment_t _incy, container_2_t _param);
 
+/**
+ * Given the Cartesian coordinates (x1, y1) of a point, the rotmg routines
+ * compute the components of a modified Givens transformation matrix H that
+ * zeros the y-component of the resulting point.
```

Comment:
Missing that the x-coord should be correctly scaled to be equal to x1. Comment applies to everywhere this description is used.

Reply:
I don't really understand what you mean here. Why does the x-coord need to be scaled?

Reply:
We are solving for $$H$$ where
```math
\begin{Bmatrix}  x  \\ y  \end{Bmatrix} 
= \begin{Bmatrix}  x_i  \\ 0 \end{Bmatrix} 
= H 
\begin{Bmatrix} x_i \sqrt{d_1}  \\ y_i \sqrt{d_2} \end{Bmatrix} 
```
where at the end $$y=0$$ and $$x=x_i$$.

You've stated the $$y=0$$ bit with *that zeros the y-component of the resulting point*, but not that we also need $$x=x_i$$.